client/src/components/backend/Drawer/Wrapper.js

```diff
@@ -2,14 +2,20 @@ import React, { PureComponent, PropTypes } from 'react';
 import { browserHistory } from 'react-router';
 import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
 import { Link } from 'react-router';
+import ScrollLock from 'components/global/Utility/ScrollLock';
```

Comment:
In general, we've been trying to import at a higher level. For example:
```
import { Utility } from 'components/global';
 
<Utility.ScrollLock>
</Utility.ScrollLock>
```

Could you do that here as well? It makes it so we don't have so many granular imports at the top of every class, which feels cleaner to me. It also makes the imports less coupled to the file system structure, which is why we have all the index.js files.